eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Mempool.hs

```diff
@@ -115,7 +118,9 @@ mempoolTransition = do
 
   -- Skip all other checks if the transaction is probably a duplicate
   whenFailureFreeDefault ledgerState $ do
-    unless (disallowUnelectedCommitteeFromVoting $ ledgerEnv ^. ledgerPpL . ppProtocolVersionL) $
+    unless
+      (hardforkConwayDisallowUnelectedCommitteeFromVoting $ ledgerEnv ^. ledgerPpL . ppProtocolVersionL)
+      $
```

Comment:
In order to avoid this ugly formatting let's float out the protocol version into its own let binding:
```suggestion
    let protVer = ledgerEnv ^. ledgerPpL . ppProtocolVersionL
    unless (hardforkConwayDisallowUnelectedCommitteeFromVoting protVer) $
```